src/search/landmarks/landmark_factory_rpg_sasp.cc

```diff
@@ -480,28 +495,30 @@ void LandmarkFactoryRpgSasp::generate_relaxed_landmarks(
 }
 
 void LandmarkFactoryRpgSasp::approximate_lookahead_orders(
-    const TaskProxy &task_proxy, const vector<vector<int>> &lvl_var, LandmarkNode *lmp) {
-    // Find all var-val pairs that can only be reached after the landmark
-    // (according to relaxed plan graph as captured in lvl_var)
-    // the result is saved in the node member variable forward_orders, and will be
-    // used later, when the phase of finding LMs has ended (because at the
-    // moment we don't know which of these var-val pairs will be LMs).
+    const TaskProxy &task_proxy, const vector<vector<bool>> &reached, LandmarkNode *lmp) {
+    /*
+      Find all var-val pairs that can only be reached after the landmark
+      (according to relaxed plan graph as captured in reached).
+      The result is saved in the node member variable forward_orders, and
+      will be used later, when the phase of finding LMs has ended (because
+      at the moment we don't know which of these var-val pairs will be LMs).
+    */
     VariablesProxy variables = task_proxy.get_variables();
-    find_forward_orders(variables, lvl_var, lmp);
+    find_forward_orders(variables, reached, lmp);
 
-    // Use domain transition graphs to find further orders. Only possible if lmp is
-    // a simple landmark.
+    // Use domain transition graphs to find further orders. Only possible
+    // if lmp is a simple landmark.
```

Comment:
This and the following three comments could be in the new multi-line style as well.